docs/user-guide/configure-zowe-runtime.md

```diff
@@ -0,0 +1,46 @@
+# Initializing Zowe z/OS runtime
```

Comment:
I believe this page is redundant and could be removed entirely for the simplicity

Reply:
The purpose of this article is that there are two options under this category (config with zwe init, and z/OSMF workflow. I see your point that it is a bit redundant, but it does help to keep the options discrete under a single category.